blog/2020-11-10-streaming-vitess-at-bolt.adoc

```diff
@@ -0,0 +1,181 @@
+= Streaming Vitess at Bolt
```

Comment:
The file name needs adjustment if we want to publish it today :)

Reply:
The file is renamed to be published today